src/libraries/System.Console/src/System/Console.cs

```diff
@@ -140,15 +128,15 @@ public static Encoding OutputEncoding
                     if (s_out != null && !s_isOutTextWriterRedirected)
                     {
                         s_out.Flush();
-                        Volatile.Write(ref s_out, null!);
+                        s_out = null;
```

Comment:
@VSadov, in our uodated memory model, does writing null to a shared location have store release semantics?